tests/test_workers/test_pkg_managers/test_general_js.py

```diff
@@ -140,6 +140,63 @@ def test_download_dependencies(
     )
 
 
+@mock.patch("tempfile.TemporaryDirectory")
+@mock.patch("os.path.exists")
+@mock.patch("cachito.workers.pkg_managers.general_js.generate_and_write_npmrc_file")
+@mock.patch("cachito.workers.pkg_managers.general_js.run_cmd")
+@mock.patch("shutil.move")
+@mock.patch("cachito.workers.paths.get_worker_config")
+def test_download_dependencies_skip_deps(
+    mock_gwc, mock_move, mock_run_cmd, mock_gawnf, mock_exists, mock_td, tmpdir,
+):
+    bundles_dir = tmpdir.mkdir("bundles")
+    mock_gwc.return_value.cachito_bundles_dir = str(bundles_dir)
+    mock_td.return_value.__enter__.return_value = "/tmp/cachito-agfdsk"
```

Comment:
Can we please use `tmpdir` here?

Reply:
I made the change, though it is not as bad as it looks since nothing writes to that directory since everything is mocked out but yes, it would be safer to use tmpdir.